### DISCUSSION TITLE
Learn DSA by an app which gives us games Like interface

### DISCUSSION THREAD

**Discussion by kajalchoudhary1003** [scaleracademy/open-source-projects#154] [Ideas]
As we all know being computer science students it is very difficult for us to learn DSA and also this topic can't be also left as it is needed to crack any company's interview or it is indeed in other coding competitions also. So I have an idea that we can make this thing become easy for students by developing a gaming interface app where they can learn the DSA from basic to advanced without being get bored and it also lets them understand easily. 
I got this idea through the app mimo which helps us to learn web development, python and many other coding languages in a very interesting way.

**Comment by 21pravi** [comment]
great idea
